packages/web3-core/src/web3_config.ts

```diff
@@ -37,6 +37,7 @@ export interface Web3ConfigOptions {
 	transactionConfirmationPollingInterval?: number;
 	blockHeaderTimeout: number;
 	maxListenersWarningThreshold: number;
+	contractDataInputFill: 'data' | 'input' | 'both'
```

Comment:
@luu-alex  This config is specific for contract package only, so imo should be in contract package only instead of in web3_config class.

Reply:
Setting it in web3_config class will allow users to only create this option once for all contracts. If this was only in the contracts package, users with multiple contracts will need to define this option for every contract instance